modules/serverless-describe-subs-kn.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-subs.adoc
+
+[id="serverless-describe-subs-kn_{context}"]
+= Describing subscriptions using the Knative CLI
+
+You can print information about a subscription in the terminal by using the `kn` CLI.
+
+.Prerequisites
+
+To create subscriptions using the `kn` CLI, ensure that:
+
+* You have installed the `kn` CLI.
+* You have created a subscription in your cluster.
+
+.Procedure
+
+* Describe a subscription:
++
+[source,terminal]
+----
+$ kn subscription describe <subscription_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:            sub1
+Namespace:       default
+Annotations:     messaging.knative.dev/creator=minikube-user, messaging.knative.dev/lastModifier=min ...
+Age:             43s
+Channel:         Channel:c0 (messaging.knative.dev/v1beta1)
```

Comment:
`c0` and `b0` are a bit untypical names for channels and broker. I would reuse names here that are used in other examples, too.